lib/std/net.zig

```diff
@@ -1177,10 +1179,10 @@ fn linuxLookupNameFromDnsSearch(
         if (byte == '.') dots += 1;
     }
 
-    const search = if (rc.search.isNull() or dots >= rc.ndots or mem.endsWith(u8, name, "."))
```

Comment:
How come this change made it in this PR?

Reply:
`rc.search.isNull()` just does `rc.search.items.len == 0` in which case `rc.search.items` would be an empty string anyways so both branches would result in the same value.

Reply:
Gotcha, thanks!